InvenTree/part/templates/part/part_sidebar.html

```diff
@@ -9,7 +9,7 @@
 {% trans "Parameters" as text %}
 {% include "sidebar_item.html" with label="part-parameters" text=text icon="fa-th-list" %}
 {% if part.is_template %}
-{% trans "Variants" as text %}
+{% trans "Variants" as text %}F
```

Comment:
I think you want to remove this "F"

![image](https://user-images.githubusercontent.com/4020546/156242682-2c3761f6-f134-4299-9174-2ffe3a1301ff.png)